public/css/concept-page.css

```
@@ -151,13 +156,38 @@
-.concept-page-b2__study-note-eyebrow { display: block; margin-bottom: 8px; }
+.concept-page-b2__study-note-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 16px;
+  margin-bottom: 8px;
+}
+.concept-page-b2__study-note-eyebrow { display: block; }
+.concept-page-b2__study-note-toggle {
+  border: 0;
+  border-bottom: 1px dotted currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)